This is a single instance where the wrong Free Software Foundation address is
used.

Index: gargoyle/src/terps/level9/bitmap.c
===================================================================
--- gargoyle.orig/src/terps/level9/bitmap.c
+++ gargoyle/src/terps/level9/bitmap.c
@@ -18,7 +18,7 @@
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
-* Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111, USA.
+* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 *
 \***********************************************************************/
 
